### PR TITLE
made neural net. 69% accuracy

### PR DESCRIPTION
**Pull request by pgrutzik**
Using the same hero pick data (create_input_data.py) as the svm. at some point, we should prob make a separate file to load data and put the models in a diff file.

### CODE REVIEW

File: neuralnet.py [hunk 1]
```diff
@@ -0,0 +1,91 @@
+# remember to reactivate tensor flow when you open a new terminal window
+
```
Comment (joehiggins) [inline neuralnet.py:2]: Agree... yea let's move all this logic to a separate file ASAP and then have separate files for each model that pull data of the same format. Good for now though.

File: neuralnet.py [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+
+########################
+#train a neural net
+from keras.models import Sequential
+from keras.layers import Dense, Activation
+
+# questions: what are units
+NUM_OUTPUT_UNITS = 1
+NUM_LAYERS = 3
```
Comment (joehiggins) [inline neuralnet.py:67]: Any reason for the number of layers? Did we try other values?
